package.json

```diff
@@ -20,7 +24,7 @@
     },
     "scripts": {
         "clean": "rm -rf out",
-        "compile": "tsc --build",
+        "compile": "tsc --build && npm run compile --workspaces --if-present",
```

Comment:
Given `types` package is inside of `language-server-runtimes` package, won't the package already contain `types` in it? Isn't it better to have `types` and `runtimes` as 2 fully separate packages?

Reply:
The way I see it we want to eventually have a similar packaging structure like `vscode-languageserver`, where we can either install the whole `language-server-runtimes` or only select packages (`types`, `protocol`, `runtimes`, etc).

Making them 2 fully separate packages (basically separate repos?), I feel will introduce more delays in development

Reply:
I do not suggest separate repository. Also, I'm not sure we'll split other folders in `language-server-runtimes` into packages. But I suggest to keep the 2 packages we are creating now (`types` and `runtime` containing all the rest) in the root of project so that they are separated. Currently, types package is inside the `runtime` package.

Reply:
afaik the root `package.json` remains private even when we want to export the whole package. It's just that there will be a separate subpackage that reexports the content from other sub packages. 
For example, here in [`vscodelanguageserver` repo the root package is private](https://github.com/microsoft/vscode-languageserver-node/blob/8e625564b531da607859b8cb982abb7cdb2fbe2e/package.json#L2)